src/component/Calculator.css

```diff
@@ -0,0 +1,66 @@
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.calculator {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 5%;
+}
+
+.display-value {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  background-color: #858694;
+  color: #fff;
+  padding-inline-end: 5px;
+  text-align: right;
+  padding: 17px;
+  font-size: 30px;
+  cursor: default;
+}
```

Comment:
- [ ] Kindly fix the design for the display part to match the provided design 

| Your design  | Provided design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/212024017-75695aec-5460-4c95-8a2b-3ec97c7f66cf.png)  | ![image](https://user-images.githubusercontent.com/53402600/212024105-53304b67-968c-4cd0-918a-cfe75c708a04.png) |